src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -69,14 +69,23 @@ public sealed class Schema : IReadOnlyList<Schema.Column>
             return null;
         }
 
+        public IEnumerable<Column> GetColumns(params string[] names)
+        {
```

Comment:
I really would rather not. What is the point of adding such a thing to `Schema`? If we wanted to have some internal extension method, that's fine.